src/esp/sim/Simulator.h

```diff
@@ -845,6 +845,20 @@ class Simulator {
       const assets::AssetInfo& assetInfo,
       const assets::RenderAssetInstanceCreationInfo& creation);
 
+  /**
+   * @brief Runs convex hull decomposition on a specified file.
+   *
+   * @param filename The MeshMetaData filename to be converted.
+   * @param CHDFilename The new filename for the CHD collision mesh.
+   * @param params VHACD params that specify resolution, vertices per convex
+   * hull, etc.
+   */
```

Comment:
Need to have @return here.  Also, need to expand comment to cover all of the functionality (", creates an object attributes referencing the convex hull asset, and returns the handle of this attributes" or something to that effect.)

Reply:
`@returns` [works too](https://www.doxygen.nl/manual/commands.html#cmdreturns), nevertheless consistency is king :)